venia-integration-tests/src/fixtures/accountAccess/index.js

```diff
@@ -0,0 +1,10 @@
+export const firstName = 'John';
+
+export const lastName = 'Doe';
+
+export const accountPassword = '123123^q';
+
+// Need unique email for test account creation
+const uuid = () => Cypress._.random(0, 1e6)
+const id = uuid()
+export const accountEmail = `${id}test@tester.com`
```

Comment:
```suggestion
export const accountEmail = `${id}test@example.com`
```
Best to take one of these https://tools.ietf.org/html/rfc2606

Reply:
thanks for the feedback @fooman , updated.